### PR TITLE
install cached android emu on jenkins workers

### PR DESCRIPTION
**Pull request by estute**
@jzoldak @benpatterson PTAL
@jibsheet 

this is a *hopefully* temporary fix for https://code.google.com/p/android/issues/detail?id=228113. I pulled the emulator off of my machine and hosted it on s3.

### CODE REVIEW

**Comment by estute** [comment]
In case this is not a good fix, the next move would be to change the emulators the mobile team uses (of course all other system images are still available ;) ), but this has caused tests to fail in the past, and I am trying to minimize work for them. Let me know what you guys think.

File: playbooks/roles/jenkins_worker/tasks/test_android_worker.yml [hunk 1]
```diff
@@ -37,3 +37,11 @@
   assert:
     that:
       - "item.stat.exists == True"
+# TEMP: until we either change to using a different system image OR google hosts
+# the android 23 image again
```
Comment (benpatterson) [inline playbooks/roles/jenkins_worker/tasks/test_android_worker.yml:41]: nit: reference the issue url that you pointed out to us in the main body of the pull request

**Comment by benpatterson** [comment]
I have a nit, but 👍 either way

**Comment by estute** [comment]
@feanil, in case Kevin is swamped

**Comment by jibsheet** [comment]
Please include the issue link per Ben's request and make an Ostrich joke.
We should consider an ansible library module for curl/checksum.

**Review by jibsheet — APPROVED** [review]

File: playbooks/roles/jenkins_worker/tasks/test_android_worker.yml (hunk 1 above)
Comment (estute) [reply playbooks/roles/jenkins_worker/tasks/test_android_worker.yml:41]: Done. added it in the actual play that does the installing

**Pull request merged**